libs/client/src/system.rs

```diff
@@ -23,4 +23,39 @@ impl KanidmClient {
         self.perform_delete_request_with_body("/v1/system/_attr/badlist_password", list)
             .await
     }
+    pub async fn system_authsession_expiry_get(&self) -> Result<u32, ClientError> {
```

Comment:
space above this line.

Reply:
I feel like you do this just to see if I'm paying attention :P

Reply:
Dang you got me /s